.github/workflows/python-package.yml

```diff
@@ -0,0 +1,37 @@
+# This workflow will install Python dependencies, run tests and lint with a variety of Python versions
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Python package
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
this might need some tweaking, but the easiest way to experiment is to land this PR as it is and push/raise Rs

Reply:
Cool. So I merged your commit, pulled and then pushed a new change (just replacing those calls to `np.random.rand` that you found in `test_control`), but I didn't see anything reflecting that the tests were run.

Reply:
hmm - if you are directly pushing to master I would expect those to run.

Although best practise is to not push directly to master, try fixing something else and creating a PR? We'll see if that runs.

Reply:
Yeah I've just been pushing because I've (embarrassingly) never really worked with separate feature branches / issuing pull requests - I've always just been pushing directly to master. I'll do that now though to test this out - it's high time I learned how to use PRs properly anyway :)

Reply:
It is honestly wonderful once you get the hang of it :-) I highly recommend reading this: https://github.com/susam/gitpr#create-pull-request - it's what I've been using for contributing to pymdp so far!

Reply:
This looks like exactly what I need, thank you! So I just created a branch, made a tiny change to the control.py file and then made a pull request. It did all the tests, as intended :) Super cool